### ISSUE TITLE
No level-one header on the sign up selector

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4017]
## Area or Component
Sign up selector

## Quick Summary
There's no level one heading on the page (no h1 element)

## Accessibility Violation
Page should contain a level-one heading. https://dequeuniversity.com/rules/axe/4.4/page-has-heading-one?application=AxeChrome

## Action Requested
Add a level one heading to the page. (The existing content of "Sign up for SimpleReport" should work)

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=543263785&range=13:13

**Comment by emmastephenson** [comment]
Fixed by #4276
